frontend/src/components/ShootWorkers/VolumeType.vue

```diff
@@ -88,16 +132,51 @@ export default {
         return 'This volume type may not be supported by your worker'
       }
       return undefined
+    },
+    isAWS () {
+      const cloudProfile = this.cloudProfileByName(this.cloudProfileName)
+      return get(cloudProfile, 'metadata.cloudProviderKind') === 'aws'
+    },
+    workerIops: {
+      get () {
+        return this.internalIops
+      },
+      set (value) {
+        this.internalIops = value
+        if (value > 0) {
```

Comment:
why not parseInt at the beginning

Reply:
If I would do that, no value would result in 0, which would lead to a validation error (minimum value) even for gp2 volumes that do not require a value at all.